### PR TITLE
app-admin/ananicy: initial import

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**
Signed-off-by: Alessandro Barbieri <lssndrbarbieri@gmail.com>
Closes: https://bugs.gentoo.org/726318

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
@thesamesam

File: app-admin/ananicy/files/ananicy.initd [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/openrc-run
+
+pidfile="/run/ananicy.pid"
+command="/usr/bin/ananicy"
+command_args="start"
+command_background=true
+
+start_pre() {
+       /sbin/sysctl -e kernel.sched_autogroup_enabled=0
```
Comment (thesamesam) [inline app-admin/ananicy/files/ananicy.initd:9]: But the path is still /sbin/ here?

File: app-admin/ananicy/files/ananicy.initd [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+stop_post() {
+       /sbin/sysctl -e kernel.sched_autogroup_enabled=1
```
Comment (thesamesam) [inline app-admin/ananicy/files/ananicy.initd:13]: Spaces for indentation?

File: app-admin/ananicy/ananicy-2.2.1-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+DESCRIPTION="ANother Auto NICe daemon"
+HOMEPAGE="https://github.com/Nefelim4ag/Ananicy"
+SRC_URI="https://github.com/Nefelim4ag/Ananicy/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P^}"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+DEPEND="${PYTHON_DEPS}"
```
Comment (thesamesam) [inline app-admin/ananicy/ananicy-2.2.1-r1.ebuild:19]:
```suggestion
BDEPEND="${PYTHON_DEPS}"
```

File: app-admin/ananicy/ananicy-2.2.1-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+PYTHON_COMPAT=( python3_{7,8,9} pypy3 )
+
+inherit python-single-r1
+
+DESCRIPTION="ANother Auto NICe daemon"
+HOMEPAGE="https://github.com/Nefelim4ag/Ananicy"
+SRC_URI="https://github.com/Nefelim4ag/Ananicy/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P^}"
+LICENSE="GPL-3"
```
Comment (thesamesam) [inline app-admin/ananicy/ananicy-2.2.1-r1.ebuild:14]:
```suggestion

LICENSE="GPL-3"
```

as per skel.ebuild

File: app-admin/ananicy/ananicy-2.2.1-r1.ebuild [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{7,8,9} pypy3 )
```
Comment (thesamesam) [inline app-admin/ananicy/ananicy-2.2.1-r1.ebuild:6]:
```suggestion
PYTHON_COMPAT=( python3_{8,9} pypy3 )
```

File: app-admin/ananicy/files/fix-sysctl-path.patch [hunk 6]
```diff
@@ -0,0 +1,14 @@
+--- a/ananicy.service	2021-04-19 19:01:54.681195153 +0200
```
Comment (thesamesam) [inline app-admin/ananicy/files/fix-sysctl-path.patch:1]:
```suggestion
--- a/ananicy.service
```

File: app-admin/ananicy/files/fix-sysctl-path.patch [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
++++ b/ananicy.service	2021-04-19 19:02:44.467050928 +0200
```
Comment (thesamesam) [inline app-admin/ananicy/files/fix-sysctl-path.patch:2]:
```suggestion
+++ b/ananicy.service
```

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: app-admin/ananicy/ananicy-2.2.1-r1.ebuild (hunk 4 above)
Comment (Alessandro-Barbieri) [reply app-admin/ananicy/ananicy-2.2.1-r1.ebuild:14]: I'm using `S` that stay in between `SRC_URI` and `LICENSE`

Comment (thesamesam) [reply app-admin/ananicy/ananicy-2.2.1-r1.ebuild:14]: I'm saying put a new line before LICENSE?

**Comment by Alessandro-Barbieri** [comment]
@thesamesam is all ok now?

**Pull request closed without merging**